graph_api/relationship/relationship_service.py

```diff
@@ -0,0 +1,70 @@
+import requests
+from requests.auth import HTTPBasicAuth
+from database_config import database
+from relationship.relationship_model import RelationshipOut
+
+
+class RelationshipService:
+    """
+    Object to handle logic of relationships requests
+
+    Attributes:
+        database_url (str): URL to used database
+        database_auth (HTTPBasicAuth): Used to authenticate to database
+    """
+    database_url = (database["address"] + database["commit_path"]) \
+        .replace("{database_name}", database["name"])
+    database_auth = HTTPBasicAuth(database["user"], database["passwd"])
+
+    def save_relationship(self, relationship):
```

Comment:
Based on #15 can be added
```suggestion
    def save_relationship(self, relationship: RelationshipIn):
```